libs/SmartStore/src/com/salesforce/androidsdk/smartstore/config/StoreConfig.java

```diff
@@ -99,6 +94,13 @@ public void registerSoups(SmartStore store) {
             try {
                 JSONObject soupConfig = soupsConfig.getJSONObject(i);
                 String soupName = soupConfig.getString(SOUP_NAME);
+
+                // Leaving soup alone if it already exists
```

Comment:
Not changing the behavior here - the registerSoup that comes a few lines later would not do anything if the soup already exists. But that way, it's clearer to the developer looking at the logs that the soup was not recreated.